sdk/messaging/azeventhubs/processor_partition_client.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package azeventhubs
+
+import "context"
+
+// ProcessorPartitionClient allows you to receive events, similar to a PartitionClient, with
+// integration into a checkpoint store for tracking progress.
+// NOTE: This type is instantiated using the Processor type, which handles dynamic load balancing.
+type ProcessorPartitionClient struct {
+	partitionID           string
+	innerClient           *PartitionClient // *azeventhubs.PartitionClient
+	checkpointStore       CheckpointStore
+	cleanupFn             func()
+	consumerClientDetails consumerClientDetails
+}
+
+// ReceiveEvents receives events until the context has expired or been cancelled.
```

Comment:
Does the `count` param mean this `receives the number of events...`?

Reply:
Fixed. Made it more clear, and also fixed PartitionClient, which has the same function.